src/input/composite_device/mod.rs

```diff
@@ -308,66 +312,66 @@ impl CompositeDevice {
                 match cmd {
                     CompositeCommand::ProcessEvent(device_id, event) => {
                         if let Err(e) = self.process_event(device_id, event).await {
-                            log::error!("Failed to process event: {:?}", e);
+                            log::error!("Failed to process event: {}", e.to_string());
```

Comment:
All of this is unnecessary and adds noise to the commit series. These types of things should be done as a completely separate "cleanup" series, probably using clippy also.